content/en/docs/collector/build-connector.md

```diff
@@ -0,0 +1,544 @@
+                    span := scopeSpan.Spans().At(k)
+                    attrs := span.Attributes()
+                    mapping := attrs.AsRaw()
+                    for key, _ := range mapping {
+                        if key == c.config.AttributeName {
+                            // create metric only if span of trace had the specific attribute
+                            metrics := pmetric.NewMetrics()
+                            return c.metricsConsumer.ConsumeMetrics(ctx, metrics)
+                        }
+                    }
+                }
+            }
+        }
+        return nil
+    }
+
+
+    // Start implements the component.Component interface.
+    func (p *connectorImp) Start(ctx context.Context, _ component.Host) error {
+        p.logger.Info("Starting spanmetrics connector")
+        return nil
+    }
+
+    // Shutdown implements the component.Component interface.
+    func (p *connectorImp) Shutdown(context.Context) error {
+        p.logger.Info("Shutting down spanmetrics connector")
+        return nil
+    }
+
+## Using the Component
+
+### Summary of Using OpenTelemetry Collector Builder:
+
+You can use the OpenTelemetry Collector Builder tool to build your code and run
+it in a collector. The Collector Builder is a tool that enables you to build
+your own OpenTelemetry Collector binary. You can add or remove components
+(receivers, processors, connectors and exporters) to suit your needs.
+
+1.  Follow the OpenTelemetry Collector Builder
+    [installation instructions](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder#installation).
+
+2.  Write a Configuration File:
+
+    Once installed, the next step is to create a configuration file
+    builder-config.yaml. This file defines the collector components you want to
+    include in your custom binary.
+
+    Here is an example of the configuration file you can use featuring your new
+    connector component:
+
+        dist:
+            name: otelcol-dev-bin
+            description: Basic OpenTelemetry Collector distribution for Developers
+            output_path: ./otelcol-dev
+            otelcol_version: 0.81.0
+
+
+        exporters:
+            - gomod:
+            go.opentelemetry.io/collector/exporter/loggingexporter v0.81.0
+
+
+        processors:
+            - gomod:
+            go.opentelemetry.io/collector/processor/batchprocessor v0.81.0
+
+
+        receivers:
+            - gomod:
+        go.opentelemetry.io/collector/receiver/otlpreceiver v0.81.0
+
+
+        connectors:
+            - gomod: github.com/exampleconnector v0.81.0
+
+
+        replaces:
+        # a list of "replaces" directives that will be part of the resulting go.mod
+
+        # This replace statement is necessary since the newly added component is not found/published to GitHub yet. Replace references to GitHub path with the local path
+        - github.com/exampleconnector => [PATH-TO-COMPONENT-CODE]/exampleconnector
```

Comment:
```suggestion
        connectors:
            - gomod: github.com/gord02/exampleconnector v0.81.0


        replaces:
        # a list of "replaces" directives that will be part of the resulting go.mod

        # This replace statement is necessary since the newly added component is not found/published to GitHub yet. Replace references to GitHub path with the local path
        - github.com/gord02/exampleconnector => [PATH-TO-COMPONENT-CODE]/exampleconnector
```